File: Common Modules/modGlobals.vb
Option Explicit On
Option Strict Off
Option Infer On
Imports System.Data
Imports System.Data.SqlClient
Imports System.Collections.Generic
Module modGlobals
    Public msdbServer As String = ""
    Public msConnection As String = ""
    Public giRank As Integer = 0
    Public giPositionID As Integer = 0
    'Public gCN As New SqlConnection
    Public gsVersion As String = ""
    Public gsDatabase As String = ""
    'Public gCN As New SqlConnection
    'Public gCNBoss As New SqlConnection
    'Public mCNLogs As New SqlConnection
    Public gCN As SqlConnection
    Public gCNBoss As SqlConnection
    Public mCNLogs As SqlConnection

    Public msConnectionBoss As String = ""
    Public gsUserID As String = ""
    Public gsRank As String = ""
    Public giSecurityID As Integer
    Public gsUserEmail As String = ""
    Public dsSecurity As DataSet
    Public gSecurityList As New List(Of SecurityMatrix)
    Public gbValidUser As Boolean = False
    Public gsCostAnalysisData As DataSet
    Public gsConnectionString As String
    Public gsPathCost As String
    Public gsUpdateFolder As String ' 9/8/23 mrb
    'Private Function FindSecurity()
    '    gSecurityList.Find(
    'End Function


End Module

File: Common Modules/modGlobals.cs
using System.Collections.Generic;
using System.Data;
using System.Data.SqlClient;

namespace BossAdmin
{
    static class modGlobals
    {
        public static string msdbServer = "";
        public static string msConnection = "";
        public static int giRank = 0;
        public static int giPositionID = 0;
        // Public gCN As New SqlConnection
        public static string gsVersion = "";
        public static string gsDatabase = "";
        // Public gCN As New SqlConnection
        // Public gCNBoss As New SqlConnection
        // Public mCNLogs As New SqlConnection
        public static SqlConnection gCN;
        public static SqlConnection gCNBoss;
        public static SqlConnection mCNLogs;

        public static string msConnectionBoss = "";
        public static string gsUserID = "";
        public static string gsRank = "";
        public static int giSecurityID;
        public static string gsUserEmail = "";
        public static DataSet dsSecurity;
        public static List<SecurityMatrix> gSecurityList = new List<SecurityMatrix>();
        public static bool gbValidUser = false;
        public static DataSet gsCostAnalysisData;
        public static string gsConnectionString;
        public static string gsPathCost;
        public static string gsUpdateFolder; // 9/8/23 mrb
                                             // Private Function FindSecurity()
                                             // gSecurityList.Find(
                                             // End Function


    }
}